src/services/thecocktailAPI.js

```diff
@@ -0,0 +1,27 @@
+export const getCategoriescocktail = () => fetch('https://www.thecocktaildb.com/api/json/v1/1/list.php?c=list')
+  .then((response) => response.json()
+  .then((json) => (response.ok ? Promise.resolve(json) : Promise.reject(json))));
+
+export const getIngredientscocktail = () => fetch('https://www.thecocktaildb.com/api/json/v1/1/list.php?i=list')
+  .then((response) => response.json()
+  .then((json) => (response.ok ? Promise.resolve(json) : Promise.reject(json))));
+
+export const getcocktailName = (cocktail) => fetch(`https://www.thecocktaildb.com/api/json/v1/1/search.php?s=${cocktail}`)
+  .then((response) => response.json()
+  .then((json) => (response.ok ? Promise.resolve(json) : Promise.reject(json))));
+
+export const getcocktailsFirstLetter = (letter) => fetch(`https://www.thecocktaildb.com/api/json/v1/1/search.php?f=${letter}`)
+  .then((response) => response.json()
+  .then((json) => (response.ok ? Promise.resolve(json) : Promise.reject(json))));
```

Comment:
em casos repetitivos como esse não acha melhor transformar tudo em uma função ? ou até mantem como funções diferentes porem que chamam uma função em comum tipo assim 
```suggestion
export const getcocktailName = (cocktail) => getCocktail('s', cocktail)

export const getcocktailsFirstLetter = (letter) => getCocktail('f', letter);

const getCocktail = (path, param) => fetch(`https://www.thecocktaildb.com/api/json/v1/1/search.php?${path}=${param}`)
  .then((response) => response.json()
  .then((json) => (response.ok ? Promise.resolve(json) : Promise.reject(json))));
```
dessa forma não apenas nesses 2 casos mas em todos que se repetem fica um codigo mais limpo